src/components/screens/SettingsScreen/SecurityScreen/ShowRecoveryPhraseScreen/ShowRecoveryPhraseScreen.tsx

```diff
@@ -49,76 +49,90 @@ const ShowRecoveryPhraseScreen: React.FC<ShowRecoveryPhraseScreenProps> = ({
   };
 
   return (
-    <BaseLayout insets={{ top: false }}>
+    <BaseLayout
+      useKeyboardAvoidingView
+      useSafeArea
+      insets={{ top: false, bottom: false }}
+    >
       <View className="flex-1">
-        <View className="bg-background-tertiary rounded-xl mt-4 mb-6">
-          <View className="p-4">
-            <View className="mb-4">
-              <Text secondary>{t("showRecoveryPhraseScreen.keepSafe")}</Text>
-            </View>
-            <View className="mb-6">
-              <Text secondary>
-                {t("showRecoveryPhraseScreen.accessWarning")}
-              </Text>
-            </View>
+        <ScrollView
+          className="flex-1"
+          showsVerticalScrollIndicator={false}
+          keyboardShouldPersistTaps="handled"
+        >
```

Comment:
@luiz-jung @danilosilvackl FYI, when we have too much content on screen, you'll probably need to do something like this in conjunction with `useKeyboardAvoidingView` so the whole view is scrolled up to display the input field.